### ISSUE TITLE
Time_zone info

### ISSUE THREAD

**Issue by dangusev** [futurecolors/django-geoip#55]
Hey guys. I think that it would be great to get info about time_zone using your application. May be you have some plans about it? Thanks!

**Comment by coagulant** [comment]
The library is stable, there are no immediate plans to add new features. 

Have you considered using pygeoip for this?

**Comment by dangusev** [comment]
Yep, I did. Your application provide much more precise data for Russia and Ukraine, unlike of MaxMind-based applications.
